nav2_smac_planner/test/test_a_star.cpp

```diff
@@ -92,10 +92,6 @@ TEST(AStarTest, test_a_star_2d)
   a_star_2.setCollisionChecker(checker.get());
   num_it = 0;
   EXPECT_THROW(a_star_2.createPath(path, num_it, tolerance), std::runtime_error);
-  a_star_2.setStart(50, 50, 0);  // invalid
```

Comment:
I think this would remove some test coverage - I think this should still be OK and cause a failure since we still can't find a valid plan, no?

Reply:
We don't throw a no valid plan inside a_star. It happen in the smac_planner. 

It shouldn't remove test coverage because that branch is removed.